raspbian/raspian-stretch-lite-setup.md

```diff
@@ -0,0 +1,131 @@
+
+### To Start, Make sure you have:
+
+* Raspberry Pi 
+
+* A micro SD card — Here we will install the Raspbian Stretch Lite
+
+* Another computer to setup the image, Wi-Fi password and SSH access.
```

Comment:
I need to add a comment for user/developer that, 

> Either Ethernet  Or WiFi,  You need some way to find IP of the raspberry machine.

Reply:
https://www.raspberrypi.org/documentation/remote-access/ip-address.md